frontend/src/app/app.tsx

```diff
@@ -27,6 +31,7 @@ const App: React.FC = () => {
 
     return (
         <>
+            <Header />
```

Comment:
I guess, we should hide header when the path is /sign-in or /sign-up

Reply:
Maybe, but I guess this functionality is for another task because I didn't have this in my task.

Reply:
Header should be only on internal pages

Reply:
because you add header to the root app

Reply:
Okay, I'll fix it